blocks/library/freeform/freeform-block.js

```diff
@@ -45,6 +45,7 @@ const FREEFORM_CONTROLS = [
 		title: wp.i18n.__( 'Convert to ordered' ),
 	},
 	{
+		showDivider: true,
```

Comment:
At a glance, it's not entirely obvious to me where the divider would be shown. To the left or right?